qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/services/diagnostics/QuteErrorCode.java

```diff
@@ -66,6 +66,8 @@ public enum QuteErrorCode implements IQuteErrorCode {
 	UnexpectedMemberTypeInCaseSection("Unexpected type `{0}` in `{1}`. Expected `{2}`."),
 	UnexpectedValueInCaseSection("Unexpected value `{0}` in `{1}`. Expected value of type `{2}`."),
 	InvalidParentInCaseSection("`{0}` section must be hosted in a #switch or #when section."),
+	UnexpectedParameterInCaseSection("Unexpected `{0}` parameter. The `{1}` operator expects only one parameter."),
+	CaseParameterRequired("A parameter or operator is required in the `#{0}` section."),

```

Comment:
I will try to find a more generic error code like

`ParameterSectionRequired("A parameter is required in the `#{0}` section.")`